src/progly/run-papertests.cc

```diff
@@ -224,16 +147,16 @@ void do_randomsearch( librados::bufferlist bl_seq, int write_type, bool debug )
     int count = 0 ;
     std::string tmp = "" ;
 
-    for( int i = 0; i < std::strlen( raw ); i++ ) {
+    for( unsigned int i = 0; i < std::strlen( raw ); i++ ) {
       tmp = tmp + raw[i] ;
       count++ ;
       if( count == 0 ) {
         continue ;
       }
-      else if( count % 6 == 0 ) {
+      else if( count % 4 == 0 ) {
         if( debug )
           std::cout << "checking tmp = " << tmp << std::endl ;
-        if( std::stoi( tmp ) == search_for )
+        if( static_cast<uint32_t>( std::stoi( tmp ) ) == search_for )
```

Comment:
Although I don't follow a style guide all the time, in general let's please not use extra spaces ( tmp ) around args or before the " ;", since that is not common and wastes extra line space.
 [if ( condition ) {  // spaces inside parentheses - rare ](https://google.github.io/styleguide/cppguide.html#Conditionals)

Reply:
Sounds good. I thought it was ok in files not intended for the main skyhook code. I'll make the edits.

Reply:
Oh you're right.  No problem then, just leave if you want.